### PR TITLE
Sponsors section of the new wecahty landing page (Section 7)

### PR DESCRIPTION
**Pull request by sajen-k**
Issue Number: #912

This PR comprises of an atomic PR of seventh section (Sponsors) for the new wechaty landing page.

Below are the images of Sponsors Section running locally

Desktop Version:
Screenshot 2021-09-08 at 9 04 58 PM

Mobile Version:
Screenshot 2021-09-08 at 9 06 23 PM

### CODE REVIEW

File: docusaurus/src/pages/sponsors-section/sponsors-section.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      ></img>
+    </div>
+    <div className={styles.sponsorsFive}>
+      <img
+        src="https://user-images.githubusercontent.com/31739964/132560519-ac0eb677-fc8b-4884-8de7-ce7c446bbeb1.png"
+        className={styles.sponsorsFiveImage}
+      ></img>
+    </div>
+    <div className={styles.sponsorsSeven}>
+      <img
```
Comment (Rohitesh-Kumar-Jain) [inline docusaurus/src/pages/sponsors-section/sponsors-section.tsx]: pls add alt text to all images, it helps improve accessibility : )

Comment (Rohitesh-Kumar-Jain) [reply docusaurus/src/pages/sponsors-section/sponsors-section.tsx]: And pls add that to all PRs, that's something I am confident you should be adding, and I am a requesting change.

**Review by Rohitesh-Kumar-Jain — CHANGES_REQUESTED** [review]
Pls, add alt text to all images, in all PRs.

File: docusaurus/src/pages/sponsors-section/sponsors-section.tsx (hunk 1 above)
Comment (sajen-k) [reply docusaurus/src/pages/sponsors-section/sponsors-section.tsx]: Done!

**Comment by sajen-k** [comment]
@Rohitesh-Kumar-Jain All the changes was made in all the corresponding PRs

**Review by huan — DISMISSED** [review]
1. Please use `.webp` format instead of others.
1. Please name the image file with meaningful name, for example: `sponsorOne.png` -> `tencent.webp`

**Comment by sajen-k** [comment]
> 1. Please use `.webp` format instead of others.
> 2. Please name the image file with meaningful name, for example: `sponsorOne.png` -> `tencent.webp`

@huan Done

File: docusaurus/src/pages/sponsors-section/sponsors-section.css [hunk 2]
```diff
@@ -0,0 +1,329 @@
...
+  align-items: center;
+  -webkit-box-pack: space-evenly;
+  -ms-flex-pack: space-evenly;
+  justify-content: space-evenly;
+  height: 200px;
+  margin: 0 9%;
+  overflow-y: hidden;
+}
+
+#sponsorsOne {
```
Comment (Rohitesh-Kumar-Jain) [inline docusaurus/src/pages/sponsors-section/sponsors-section.css]: Pls do change this from sponsoring to sponsor's name, 

In general, if you have to named your variables like string1, string2, you should consider renaming them.

File: docusaurus/src/pages/sponsors-section/sponsors-section.css [hunk 3]
```diff
@@ -0,0 +1,329 @@
...
+  left: 10%;
+  top: 125px;
+}
+
+#sponsorsSectionWrapper {
+  width: 100%;
+  margin-top: 220px;
+}
+
+#sponsorsFirstSection {
```
Comment (Rohitesh-Kumar-Jain) [inline docusaurus/src/pages/sponsors-section/sponsors-section.css]: here also can you come up with a more descriptive name?

File: docusaurus/src/pages/sponsors-section/sponsors-section.css [hunk 4]
```diff
@@ -0,0 +1,329 @@
...
+  overflow-y: hidden;
+  width: 150px;
+  height: 150px;
+  background: #f4fff2;
+  border-radius: 20px;
+  -webkit-transform: rotate(-45deg);
+  transform: rotate(-45deg);
+}
+
+#sponsorsTwo {
```
Comment (Rohitesh-Kumar-Jain) [inline docusaurus/src/pages/sponsors-section/sponsors-section.css]: same as above

File: docusaurus/src/pages/sponsors-section/sponsors-section.css [hunk 5]
```diff
@@ -0,0 +1,329 @@
...
+  overflow-y: hidden;
+  width: 150px;
+  height: 150px;
+  background: #f4fff2;
+  border-radius: 20px;
+  -webkit-transform: rotate(-45deg);
+  transform: rotate(-45deg);
+}
+
+#sponsorsOneImage {
```
Comment (Rohitesh-Kumar-Jain) [inline docusaurus/src/pages/sponsors-section/sponsors-section.css]: here as well

File: docusaurus/src/pages/sponsors-section/sponsors-section.css [hunk 6]
```diff
@@ -0,0 +1,329 @@
...
+  }
+  #sponsorsThree {
+    width: 70px;
+    height: 70px;
+  }
+  #sponsorsFour {
+    width: 70px;
+    height: 70px;
+  }
+  #sponsorsFive {
```
Comment (Rohitesh-Kumar-Jain) [inline docusaurus/src/pages/sponsors-section/sponsors-section.css]: same as above

**Review by Rohitesh-Kumar-Jain — CHANGES_REQUESTED** [review]

File: docusaurus/src/pages/sponsors-section/sponsors-section.css (hunk 2 above)
Comment (sajen-k) [reply docusaurus/src/pages/sponsors-section/sponsors-section.css]: Done, Thank you for the comments.

File: docusaurus/src/pages/sponsors-section/sponsors-section.css (hunk 4 above)
Comment (sajen-k) [reply docusaurus/src/pages/sponsors-section/sponsors-section.css]: done

File: docusaurus/src/pages/sponsors-section/sponsors-section.css (hunk 5 above)
Comment (sajen-k) [reply docusaurus/src/pages/sponsors-section/sponsors-section.css]: done

File: docusaurus/src/pages/sponsors-section/sponsors-section.css (hunk 6 above)
Comment (sajen-k) [reply docusaurus/src/pages/sponsors-section/sponsors-section.css]: done

File: docusaurus/src/pages/sponsors-section/sponsors-section.css (hunk 3 above)
Comment (sajen-k) [reply docusaurus/src/pages/sponsors-section/sponsors-section.css]: ok I will look into it and we have divided the sponsors to 2 section where one contains few sponsors logo and 2nd section contains others and each of these wrapped into 2 `<div>` tags that's the reason we have given it as firstsection and secondsection.

Comment (Rohitesh-Kumar-Jain) [reply docusaurus/src/pages/sponsors-section/sponsors-section.css]: If you aren't able to come up with a meaningful name, at least add comments on top of that giving some details about it

Comment (sajen-k) [reply docusaurus/src/pages/sponsors-section/sponsors-section.css]: Done, thank you.

**Review by Rohitesh-Kumar-Jain — APPROVED** [review]
Thanks for implementing both my and @huan's suggested changes.

**Review by arnabiscreating — APPROVED** [review]
Looks good to me.

**Review by proudofsimin — APPROVED** [review]

**Pull request merged**